metacatalog/models/variable.py

```diff
@@ -152,7 +152,7 @@ def to_dict(self, deep=False) -> dict:
             symbol=self.symbol,
             unit=self.unit.to_dict(deep=False),
             column_names=self.column_names,
-            keyword=self.keyword.to_dict(deep=False)
+            keyword=self.keyword.to_dict(deep=False) if self.keyword else None
```

Comment:
I guess this is not the best option, because we have `None` in the result dict.

Usually, I take the approach to never add a non-existing key to the dictionary in the first place. Something like:

```Python
d = dict(
   ...
  column_names=self.column_names
)

if self.keywords:
  d['keyword'] = self.keyword.to_dict()
```

Reply:
Yes, that's better!  
I also replaced the old keyword code that was not working, as it did not add the keyword to the dict.